src/components/app/TabBar.css

```diff
@@ -17,20 +17,23 @@
 
 .tabBarTabWrapper {
   display: flex;
+  overflow: auto hidden;
   min-width: 0; /* This makes the tab container actually shrinkable below min-content */
+  flex: 1;
   flex-flow: row nowrap;
   padding: 0;
-  margin: 0 -0.5px; /* This combines with the -0.5px horizontal margin of .tabBarTab to clip off the first tab's 1px starting border */
+  margin: 0 -0.5px -1px; /* This combines with the -0.5px horizontal margin of .tabBarTab to clip off the first tab's 1px starting border */
   list-style: none;
+  scrollbar-width: none;
 }
 
 .tabBarTab {
   position: relative;
   min-width: 8em;
-  padding: 6px 4px;
+  padding: 6px 4px 5px;
   border: solid transparent;
   border-width: 0 1px;
-  margin: 0 -0.5px; /* This makes the 1px border between adjacent tabs overlap */
+  margin: 0 -0.5px 1px; /* This makes the 1px border between adjacent tabs overlap */
```

Comment:
New comment:

`The -0.5px horizontal margin makes the 1px border between adjacent tabs overlap. The 1px bottom margin makes space for the .Details-top-bar bottom border.`